src/main/resources/com/adobe/epubcheck/messages/MessageBundle.properties

```diff
@@ -293,6 +293,7 @@ RSC_019=EPUBs with Multiple Renditions should contain a META-INF/metadata.xml fi
 RSC_020='%1$s' is not a valid URI.
 RSC_021=A Search Key Map Document must point to Content Documents ('%1$s' was not found in the spine).
 RSC_022=Cannot check image details (requires Java version 7 or higher).
+RSC_023='%1$s' is not a valid URL.
```

Comment:
Okay, technically it's not the host component which is missing. The `URI` parser just can't identify it.

Technically it's just a missing `/` or two. That's the only cases where `RSC_023` is triggered. For `https:/www.yout` and for `https:www.yout`.

All the other cases are covered by RSC_020 or HTM_025.

So I suggest we just say exactly that: There's a missing slash after the protocol/schema of the URL. protocol is probably more known to non-geeks, right?

```
RSC_023=The URL '%1$s' is missing one or two slashes '/' after the protocol '%2$s:'
> The URL 'https:/www.youtube.com' is missing one or two slashes '/' after the protocol 'https:'
```

We could even go further and say:

```
RSC_023=The URL '%1$s' is missing %2$d slash(es) '/' after the protocol '%3$s:'
> The URL 'https:/www.youtube.com' is missing 1 slash(es) '/' after the protocol 'https:'
```

What do you think?